asmcomp/thread_sanitizer.mli

```diff
@@ -0,0 +1,37 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                      Anmol Sahoo, Purdue University                    *)
```

Comment:
I think we can get rid of this header and put the actual author names here.

Reply:
The convention seems to be that the original authors’ names are the ones put in the top comment, even if the file is modified by others. I am not certain about this but that’s the rule I applied.

Reply:
Taken to the extreme, this means that all compiler files can only have one author, namely the original creator.
I think the convention is that the original authors of the first published version are listed.
I suggest listing the three of you.

Reply:
Done, thank you for the suggestion.